packager/signer/signer_test.go

```diff
@@ -1035,3 +1035,24 @@ func (this *SignerSuite) TestPrometheusMetricGatewayRequestsLatency() {
 	}
 
 }
+
+func (this *SignerSuite) TestIfCappedDontSignAndProxyFullDocument() {
+	urlSets := []util.URLSet{{
+		Sign: &util.URLPattern{[]string{"https"}, "", this.httpsHost(), stringPtr("/amp/.*"), []string{}, stringPtr(""), false, 2000, nil}}}
+
+	const uncappedTailLength = 100
+	veryLongArray := [maxBodyLength + uncappedTailLength]string{}
+	veryLongString := strings.Join(veryLongArray[:], "a")
```

Comment:
I think you can replace this with a call to https://golang.org/pkg/strings/#Repeat.